src/app/lib/styledGrid/index.jsx

```diff
@@ -27,6 +29,155 @@ import {
   gridContainerLargeCss,
 } from '../layoutGrid';
 
+export const MediumContentContainer = styled.div`
+  @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
+    margin: 0 auto;
+    max-width: 46.4rem;
+  }
+`;
+
+const defaultColumns = {
+  group0: 6,
+  group1: 6,
+  group2: 6,
+  group3: 6,
+  group4: 6,
+  group5: 12,
+};
+
+const mediumColumns = {
+  ...defaultColumns,
+  group3: 5,
+  group4: 5,
+  group5: 10,
+};
+
+const noMargins = {
+  group0: false,
+  group1: false,
+  group2: false,
+  group3: false,
+  group4: false,
+  group5: false,
+};
+
+export const GridWrapperMedium = ({ children }) => {
+  return (
+    <Grid
+      enableGelGutters
+      parentColumns={defaultColumns}
+      columns={defaultColumns}
+    >
+      {children}
+    </Grid>
+  );
+};
+
+GridWrapperMedium.propTypes = {
+  children: node.isRequired,
+};
+
+export const GridItem = ({ children, enableMargins }) => {
+  const gridMargins = {
+    group0: true,
+    group1: true,
+    group2: true,
+    group3: true,
+    group4: false,
+    group5: false,
+  };
+
+  const margins = enableMargins ? gridMargins : noMargins;
+
+  return (
+    <Grid
+      item
+      parentColumns={defaultColumns}
+      columns={mediumColumns}
+      margins={margins}
+    >
+      {children}
+    </Grid>
+  );
+};
+
+GridItem.propTypes = {
+  children: node.isRequired,
+  enableMargins: bool,
+};
+
+GridItem.defaultProps = {
+  enableMargins: false,
+};
+
+export const GridItemSmall = ({ children }) => {
+  const columns = {
+    group0: 4,
+    group1: 4,
+    group2: 4,
+    group3: 4,
+    group4: 4,
+    group5: 8,
+  };
+
+  const margins = {
+    group0: true,
+    group1: true,
+    group2: true,
+    group3: false,
```

Comment:
I think this should be true, since when margins are applied, they should work up to and including group3:
 ```suggestion
    group3: true,
```
This can make it simpler so grid margins can be shared between `GridItemLarge`, `GridItemSmall` & `GridItem`